activation/handler.go

```diff
@@ -353,7 +358,7 @@ func (h *Handler) storeAtx(ctx context.Context, atx *types.VerifiedActivationTx)
 		return fmt.Errorf("checking if node is malicious: %w", err)
 	}
 	var proof *types.MalfeasanceProof
-	if err = h.cdb.WithTx(ctx, func(dbtx *sql.Tx) error {
+	if err := h.cdb.WithTx(ctx, func(dbtx *sql.Tx) error {
```

Comment:
curious why you want to overwrite these?

Reply:
No particular reason, I just like to limit the scope of a variable to smaller blocks if possible.